.github/workflows/Build.yml

```
@@ -35,7 +35,7 @@ jobs:
 
     - name: Calculate environment variables
       run: |     
-        $IsPrerelease = !($Env:GIT_REF -match 'release-[0-9]?\.[0-9]?$')
+        $IsPrerelease = !($Env:GIT_REF -match 'release-[0-9]+?\.[0-9]+?$')
```

```suggestion
        $IsPrerelease = !($Env:GIT_REF -match 'release-[0-9]+(?:\.[0-9]+)?$')
```
Are the lazy qualifiers intended? Because I think the greedy ones would work just fine. Additionally, I proposing a change to accept names such as "release-1" as release branch names